.vscode/notebooks/CITestResults.ipynb

```diff
@@ -274,14 +282,45 @@
     "failures = testData.where(testData['status'] == 'failed').dropna()\n",
     "failures = failures[['date', 'status', 'scenario', 'runUrl']].sort_values(by=['date'], ascending=False).head(10)\n",
     "\n",
+    "failureMessage = ''\n",
     "for index, row in failures.iterrows():\n",
-    "    print(f\"{row['date']} - {row['scenario']}\\n{row['runUrl']}\")"
+    "    print(f\"{row['date']} - {row['scenario']}\\n{row['runUrl']}\")\n",
+    "    failureMessage += f\"{row['date']} - {row['scenario']}\\n{row['runUrl']}\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import io\n",
+    "from urllib import request\n",
+    "\n",
+    "# post to create new github issue\n",
```

Comment:
The rest of this notebook could be run without any side effects, but this would create a new issue each time right?
Kai ran all cells when he tried this notebook out. We probably want some confirmation to actually create the issue

Reply:
Yeah I agree

Reply:
@amunger added a widget for confirmation before creating issue so the notebook can be executed without creating issues unexpectedly.